code/WorkInProgress/recycling/conveyor.dm

```diff
@@ -384,98 +384,91 @@
 // the conveyor control switch
 //
 //
-
-/obj/machinery/conveyor_switch
+#define CONVEYOR_FORWARD 1
+#define CONVEYOR_REVERSE -1
+#define CONVEYOR_STOPPED 0
+/obj/submachine/conveyor_switch
```

Comment:
The readme for submachines (in the code/obj/submachine folder anyway) is `// TODO: reincorporate all these as actual machines, but take them out of the process loop, if need be`

I don't know how old or accurate that bit is, but I figure if you're doing the opposite it's probably worth addressing.

Reply:
Easy enough